k8s/demo/common/money-claims/mailhog/mailhog.yaml

```diff
@@ -0,0 +1,73 @@
+# helm repo add codecentric https://codecentric.github.io/helm-charts
+# helm install --name mailhog --namespace namespace stable/mailhog
+# helm get mailhog > mailhog.yaml
+---
+# Source: mailhog/templates/service.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: mailhog
+  namespace: money-claims
+  labels:
+    app: mailhog
+spec:
+  type: "ClusterIP"
+  clusterIP: ""
+  ports:
+    - name: http
+      port: 8025
+      protocol: TCP
+      targetPort: http
+    - name: smtp
+      port: 1025
+      protocol: TCP
+      targetPort: smtp
+  selector:
+    app: mailhog
+    release: mailhog
+---
+# Source: mailhog/templates/deployment.yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: mailhog
+  namespace: money-claims
+  labels:
+    app: mailhog
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.mailhog: regexp:^v1.*
```

Comment:
dont think semver will work with the `v` 🤔

Reply:
dunno, I wouldn't worry too much